web_src/css/repo/header.css

```
@@ -1,4 +1,8 @@
-.header-wrapper .fork-flag {
+.repository .secondary-nav {
+  padding-top: 12px;
+}
+
+.secondary-nav .fork-flag {
```

Oops, I misread, I think your old code is better ....

```suggestion
.repository .secondary-nav .fork-flag {
```